### ISSUE TITLE
Fastlane Deliver Unexpected error (due to new language not being enabled)

### ISSUE THREAD

**Issue by frrijn** [fastlane/fastlane#10475]
### New Issue Checklist

- ✅  Updated fastlane to the latest version
- ✅  I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- ✅  I read [docs.fastlane.tools](https://docs.fastlane.tools)
- ✅  I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description

Unexpected Error when uploading metadata to ITC.

Had the same error 12 hours ago. (playing the waiting game did not solve the problem)

ITC is/was online according to apple

Uploading for testflight works completely fine.

##### Complete output when running fastlane, including the stack trace and command used
> You can use: `--capture_output` as the last commandline argument to get that collected for you

```
---iMac:-- --$ bundle exec fastlane deliver --force
[15:01:17]: Successfully loaded '/Users/--/Documents/--/fastlane/Deliverfile' 📄

+-----------------------+-----------------------+
| Detected Values from './fastlane/Deliverfile' |
+-----------------------+-----------------------+
| app_identifier        | --.--            |
| username              | --@--.--        |
+-----------------------+-----------------------+

[15:01:17]: Running precheck before submitting to review, if you'd like to disable this check you can set run_precheck_before_submit to false
[15:01:17]: Login to iTunes Connect (--@--.--)
[15:01:19]: Login successful

+--------------------------------------+------------------------+
|                    deliver 2.59.0 Summary                     |
+--------------------------------------+------------------------+
| force                                | true                   |
| app_identifier                       | --.--             |
| username                             | --@--.--         |
| screenshots_path                     | ./fastlane/screenshots |
| metadata_path                        | ./fastlane/metadata    |
| app_version                          | 1.0.0                  |
| edit_live                            | false                  |
| ipa                                  | --.ipa             |
| platform                             | ios                    |
| skip_binary_upload                   | false                  |
| skip_screenshots                     | false                  |
| skip_metadata                        | false                  |
| skip_app_version_update              | false                  |
| submit_for_review                    | false                  |
| automatic_release                    | false                  |
| dev_portal_team_id                   | --             |
| overwrite_screenshots                | false                  |
| run_precheck_before_submit           | true                   |
| precheck_default_rule_level          | warn                   |
| ignore_language_directory_validation | false                  |
+--------------------------------------+------------------------+

[15:01:20]: Making sure the latest version on iTunes Connect matches '1.0.0' from the ipa file...
[15:01:21]: '1.0.0' is the latest version on iTunes Connect
[15:01:21]: Framed screenshots are detected! 🖼 Non-framed screenshot files may be skipped. 🏃
[15:01:21]: 🏃 Skipping screenshot file: ./fastlane/screenshots/nl-NL/iPhone 5-1_Agenda.png
[15:01:21]: 🏃 Skipping screenshot file: ./fastlane/screenshots/nl-NL/iPhone 5-2_DetailAgenda.png
[15:01:21]: 🏃 Skipping screenshot file: ./fastlane/screenshots/nl-NL/iPhone 5-3_NewHour.png
[15:01:21]: 🏃 Skipping screenshot file: ./fastlane/screenshots/nl-NL/iPhone 5-4_TaskView.png
[15:01:21]: 🏃 Skipping screenshot file: ./fastlane/screenshots/nl-NL/iPhone 5-5_ClientView.png
[15:01:21]: 🏃 Skipping screenshot file: ./fastlane/screenshots/nl-NL/iPhone 5-6_SettingsView.png
[15:01:21]: 🏃 Skipping screenshot file: ./fastlane/screenshots/nl-NL/iPhone 6 Plus-1_Agenda.png
[15:01:21]: 🏃 Skipping screenshot file: ./fastlane/screenshots/nl-NL/iPhone 6 Plus-2_DetailAgenda.png
[15:01:21]: 🏃 Skipping screenshot file: ./fastlane/screenshots/nl-NL/iPhone 6 Plus-3_NewHour.png
[15:01:21]: 🏃 Skipping screenshot file: ./fastlane/screenshots/nl-NL/iPhone 6 Plus-4_TaskView.png
[15:01:21]: 🏃 Skipping screenshot file: ./fastlane/screenshots/nl-NL/iPhone 6 Plus-5_ClientView.png
[15:01:21]: 🏃 Skipping screenshot file: ./fastlane/screenshots/nl-NL/iPhone 6 Plus-6_SettingsView.png
[15:01:21]: 🏃 Skipping screenshot file: ./fastlane/screenshots/nl-NL/iPhone 6-1_Agenda.png
[15:01:21]: 🏃 Skipping screenshot file: ./fastlane/screenshots/nl-NL/iPhone 6-2_DetailAgenda.png
[15:01:21]: 🏃 Skipping screenshot file: ./fastlane/screenshots/nl-NL/iPhone 6-3_NewHour.png
[15:01:21]: 🏃 Skipping screenshot file: ./fastlane/screenshots/nl-NL/iPhone 6-4_TaskView.png
[15:01:21]: 🏃 Skipping screenshot file: ./fastlane/screenshots/nl-NL/iPhone 6-5_ClientView.png
[15:01:21]: 🏃 Skipping screenshot file: ./fastlane/screenshots/nl-NL/iPhone 6-6_SettingsView.png
[15:01:21]: Loading './fastlane/metadata/nl-NL/description.txt'...
[15:01:21]: Loading './fastlane/metadata/nl-NL/keywords.txt'...
[15:01:21]: Loading './fastlane/metadata/nl-NL/release_notes.txt'...
[15:01:21]: Loading './fastlane/metadata/nl-NL/support_url.txt'...
[15:01:21]: Loading './fastlane/metadata/nl-NL/marketing_url.txt'...
[15:01:21]: Loading './fastlane/metadata/nl-NL/promotional_text.txt'...
[15:01:21]: Loading './fastlane/metadata/nl-NL/name.txt'...
[15:01:21]: Loading './fastlane/metadata/nl-NL/subtitle.txt'...
[15:01:21]: Loading './fastlane/metadata/nl-NL/privacy_url.txt'...
[15:01:21]: Loading './fastlane/metadata/copyright.txt'...
[15:01:21]: Loading './fastlane/metadata/primary_category.txt'...
[15:01:21]: Loading './fastlane/metadata/secondary_category.txt'...
[15:01:21]: Loading './fastlane/metadata/primary_first_sub_category.txt'...
[15:01:21]: Loading './fastlane/metadata/primary_second_sub_category.txt'...
[15:01:21]: Loading './fastlane/metadata/secondary_first_sub_category.txt'...
[15:01:21]: Loading './fastlane/metadata/secondary_second_sub_category.txt'...
[15:01:21]: Loading './fastlane/metadata/trade_representative_contact_information/trade_name.txt'...
[15:01:21]: Loading './fastlane/metadata/trade_representative_contact_information/address_line1.txt'...
[15:01:21]: Loading './fastlane/metadata/trade_representative_contact_information/city_name.txt'...
[15:01:21]: Loading './fastlane/metadata/trade_representative_contact_information/country.txt'...
[15:01:21]: Loading './fastlane/metadata/trade_representative_contact_information/postal_code.txt'...
[15:01:21]: Loading './fastlane/metadata/trade_representative_contact_information/is_displayed_on_app_store.txt'...
[15:01:21]: Loading './fastlane/metadata/review_information/first_name.txt'...
[15:01:21]: Loading './fastlane/metadata/review_information/last_name.txt'...
[15:01:21]: Loading './fastlane/metadata/review_information/phone_number.txt'...
[15:01:21]: Loading './fastlane/metadata/review_information/email_address.txt'...
[15:01:21]: Loading './fastlane/metadata/review_information/demo_user.txt'...
[15:01:21]: Loading './fastlane/metadata/review_information/demo_password.txt'...
[15:01:21]: Loading './fastlane/metadata/review_information/notes.txt'...
[15:01:23]: Activating language nl-NL...
[15:01:25]: Uploading metadata to iTunes Connect
Potential server error received: 'Unexpected Error'. Retrying after 10 seconds (remaining: 5)...
Potential server error received: 'Unexpected Error'. Retrying after 10 seconds (remaining: 5)...

Looking for related GitHub issues on fastlane/fastlane...

➡️  Unexpected Error when submitting the app for review
    https://github.com/fastlane/fastlane/issues/8819 [closed] 4 💬
    5 weeks ago

➡️  pilot without skip_waiting:true appears to be broken(?)
    https://github.com/fastlane/fastlane/issues/8867 [closed] 13 💬
    4 weeks ago

➡️  Deliver: Unexpected Error when selecting build
    https://github.com/fastlane/fastlane/issues/7757 [closed] 3 💬
    06 Apr 2017

and 10 more at: https://github.com/fastlane/fastlane/search?q=The%20request%20could%20not%20be%20completed%20because:%0A%09Unexpected%20Error&type=Issues&utf8=✓

🔗  You can ⌘ + double-click on links to open them directly in your browser.

[!] The request could not be completed because:
	Unexpected Error
```

### Environment

Please run `fastlane env` and copy the output below. This will help us help you :+1:
If you used `--capture_output` option please remove this block - as it is already included there.

[INSERT OUTPUT HERE]
<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                       |
| --------------------------- | ------------------------------------------- |
| OS                          | 10.12.6                                     |
| Ruby                        | 2.4.1                                       |
| Bundler?                    | true                                        |
| Git                         | git version 2.13.5 (Apple Git-94)           |
| Installation Source         | /usr/local/bin/fastlane                     |
| Host                        | Mac OS X 10.12.6 (16G29)                    |
| Ruby Lib Dir                | /usr/local/Cellar/ruby/2.4.1_1/lib          |
| OpenSSL Version             | OpenSSL 1.0.2k  26 Jan 2017                 |
| Is contained                | false                                       |
| Is homebrew                 | false                                       |
| Is installed via Fabric.app | false                                       |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/ |
| Xcode Version               | 9.0                                         |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   | en_US.UTF-8 | ✅ |
| LANGUAGE |             |   |

### fastlane files:

<details><summary>`./fastlane/Fastfile`</summary>

```ruby
	# Customise this file, documentation can be found here:
# https://github.com/fastlane/fastlane/tree/master/fastlane/docs
# All available actions: https://docs.fastlane.tools/actions
# can also be listed using the `fastlane actions` command

# Change the syntax highlighting to Ruby
# All lines starting with a # are ignored when running `fastlane`

# If you want to automatically update fastlane if a new version is available:
# update_fastlane

# This is the minimum version number required.
# Update this, if you use features of a newer version
fastlane_version "2.47.0"

default_platform :ios

platform :ios do
  before_all do
    ENV["SLACK_URL"] = "--"
    cocoapods

    FASTLANE_APPLE_APPLICATION_SPECIFIC_PASSWORD="--"
    
  end

  desc "Runs all the tests"
  lane :test do
    scan
  end

  # 1
  desc "Creating a code signing certificate and provisioning profile"
  # 2
  lane :provision do
    # 3
    produce(
      app_name: '--',
      language: '--',
      app_version: '--',
      sku: '--'
    )
    # 4
    cert
    # 5
    sigh(force: true)
  end

  desc "Take screenshots"
  lane :snapshot_f do
    snapshot
    frameit(white: true)
  end

  desc "increment build"
  lane :inc_build do
    increment_build_number
  end

  desc "increment patch"
  lane :inc_patch do
    increment_version_number(
      bump_type: "patch"
    )
  end

  desc "increment minor"
  lane :inc_minor do
    increment_version_number(
      bump_type: "minor"
    )
  end

  desc "increment major"
  lane :inc_major do
    increment_version_number(
      bump_type: "major"
    )
  end


  desc "Create ipa"
  lane :build do
    gym(scheme: "--",
      workspace: "--.xcworkspace",
      include_bitcode: true)
  end

  desc "Upload to App Store"
  lane :upload_appstore do
    deliver
  end

  desc "Upload to TestFlight"
  lane :upload_testflight do
    pilot
  end

  desc "Upload metadata"
  lane :upload_metadata do
    deliver(metadata_only: true)
  end

  desc "Upload sentry"
  lane :upload_symbols do
  download_dsyms
  sentry_upload_dsym(
    auth_token: '--',
    org_slug: '--',
    project_slug: '--'
  )
  end

  desc "Provision, take screenshots, build and upload to testflight"
  lane :all_testflight do
    provision
    # screenshot
    inc_build
    build
    upload_testflight
    upload_symbols
    clean_build_artifacts

    slack(
      message: "Successfully deployed testflight!"
    )

  end

  desc "Provision, take screenshots, build and upload to App Store"
  lane :all_appstore do
    provision
    # screenshot
    build
    upload_appstore
    upload_symbols
    clean_build_artifacts

    slack(
      message: "Successfully deployed new app update to the App Store!"
    )

  end

  error do |lane, exception|
    slack(
      message: exception.message,
      success: false
    )
  end
end


# More information about multiple platforms in fastlane: https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Platforms.md
# All available actions: https://docs.fastlane.tools/actions

# fastlane reports which actions are used. No personal data is recorded. 
# Learn more at https://github.com/fastlane/fastlane#metrics

```
</details>

<details><summary>`./fastlane/Appfile`</summary>

```ruby
app_identifier "--" # The bundle identifier of your app
apple_id "--" # Your Apple email address

team_id "--" # Developer Portal Team ID

# you can even provide different app identifiers, Apple IDs and team names per lane:
# More information: https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Appfile.md

```
</details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.59.0  | ✅ Up-To-Date  |

### Loaded fastlane plugins:

| Plugin                 | Version | Update-Status |
| ---------------------- | ------- | ------------- |
| fastlane-plugin-sentry | 1.4.0   | ✅ Up-To-Date  |

<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| did_you_mean              | 1.1.2        |
| bundler                   | 1.15.4       |
| io-console                | 0.4.6        |
| CFPropertyList            | 2.3.5        |
| i18n                      | 0.8.6        |
| minitest                  | 5.10.3       |
| thread_safe               | 0.3.6        |
| tzinfo                    | 1.2.3        |
| activesupport             | 4.2.9        |
| public_suffix             | 2.0.5        |
| addressable               | 2.5.2        |
| babosa                    | 1.0.2        |
| claide                    | 1.0.2        |
| fuzzy_match               | 2.0.4        |
| nap                       | 1.1.0        |
| cocoapods-core            | 1.3.1        |
| cocoapods-deintegrate     | 1.0.1        |
| cocoapods-downloader      | 1.1.3        |
| cocoapods-plugins         | 1.0.0        |
| cocoapods-search          | 1.0.0        |
| cocoapods-stats           | 1.0.0        |
| netrc                     | 0.7.8        |
| cocoapods-trunk           | 1.2.0        |
| cocoapods-try             | 1.1.0        |
| colored2                  | 3.1.2        |
| escape                    | 0.0.4        |
| fourflusher               | 2.0.1        |
| gh_inspector              | 1.0.3        |
| molinillo                 | 0.5.7        |
| ruby-macho                | 1.1.0        |
| nanaimo                   | 0.2.3        |
| xcodeproj                 | 1.5.2        |
| cocoapods                 | 1.3.1        |
| colored                   | 1.2          |
| highline                  | 1.7.8        |
| commander-fastlane        | 4.4.5        |
| declarative               | 0.0.10       |
| declarative-option        | 0.1.0        |
| unf_ext                   | 0.0.7.4      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20170404 |
| dotenv                    | 2.2.1        |
| excon                     | 0.59.0       |
| multipart-post            | 2.0.0        |
| faraday                   | 0.13.1       |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.6        |
| faraday_middleware        | 0.12.2       |
| fastimage                 | 2.1.0        |
| jwt                       | 1.5.6        |
| little-plugger            | 1.1.4        |
| multi_json                | 1.12.2       |
| logging                   | 2.2.2        |
| memoist                   | 0.16.0       |
| os                        | 0.9.6        |
| signet                    | 0.7.3        |
| googleauth                | 0.5.3        |
| httpclient                | 2.8.3        |
| mime-types-data           | 3.2016.0521  |
| mime-types                | 3.1          |
| uber                      | 0.1.0        |
| representable             | 3.0.4        |
| retriable                 | 3.1.1        |
| google-api-client         | 0.13.6       |
| json                      | 2.1.0        |
| mini_magick               | 4.5.1        |
| multi_xml                 | 0.6.0        |
| plist                     | 3.3.0        |
| rubyzip                   | 1.2.1        |
| security                  | 0.1.3        |
| slack-notifier            | 1.5.1        |
| terminal-notifier         | 1.8.0        |
| unicode-display_width     | 1.3.0        |
| terminal-table            | 1.8.0        |
| tty-screen                | 0.5.0        |
| word_wrap                 | 1.0.0        |
| rouge                     | 2.0.7        |
| xcpretty                  | 0.2.8        |
| xcpretty-travis-formatter | 0.0.4        |
| fastlane-plugin-sentry    | 1.4.0        |
</details>

*generated on:* **2017-10-02**
</details>

**Comment by frrijn** [comment]
[15:24:42]: ------------------
[15:24:42]: --- Step: cert ---
[15:24:42]: ------------------

+---------------+----------------------------------------------------+
|                      Summary for cert 2.59.0                       |
+---------------+----------------------------------------------------+
| development   | false                                              |
| force         | false                                              |
| username      | --@--.--                                    |
| team_id       | --                                        |
| keychain_path | /Users/--/Library/Keychains/login.keyc  |
|               | hain-db                                            |
| platform      | ios                                                |
+---------------+----------------------------------------------------+

[15:24:42]: Starting login with user '--@--.--'
[15:24:43]: Successfully logged in
[15:24:44]: Found the certificate -- (iOS Distribution) which is installed on the local machine. Using this one.
[15:24:44]: Verifying the certificate is properly installed locally...
[15:24:44]: Successfully installed certificate --
[15:24:44]: Use signing certificate '--' from now on!
[15:24:44]: ------------------
[15:24:44]: --- Step: sigh ---
[15:24:44]: ------------------

+-------------------------------------+----------------+
|               Summary for sigh 2.59.0                |
+-------------------------------------+----------------+
| force                               | true           |
| adhoc                               | false          |
| development                         | false          |
| skip_install                        | false          |
| app_identifier                      | --.--     |
| username                            | --@--.-- |
| team_id                             | --     |
| ignore_profiles_with_different_name | false          |
| cert_id                             | --     |
| skip_fetch_profiles                 | false          |
| skip_certificate_verification       | false          |
| platform                            | ios            |
+-------------------------------------+----------------+

[15:24:44]: Starting login with user '--@--.--'
[15:24:47]: Successfully logged in
[15:24:47]: Fetching profiles...
[15:24:48]: Verifying certificates...
[15:24:49]: Found 1 matching profile(s)
[15:24:49]: Recreating the profile
[15:24:51]: Creating new provisioning profile for '--.--' with name '--.-- AppStore' for 'ios' platform
[15:24:52]: Downloading provisioning profile...
[15:24:53]: Successfully downloaded provisioning profile...
[15:24:53]: Installing provisioning profile...
/Users/--/Documents/--/AppStore_net.--.mobileprovision
[15:24:53]: Setting Provisioning Profile type to 'app-store'
[15:24:53]: Cruising back to lane 'ios all_appstore' 🚘
[15:24:53]: --------------------------------------
[15:24:53]: --- Step: Switch to ios build lane ---
[15:24:53]: --------------------------------------
[15:24:53]: Cruising over to lane 'ios build' 🚖

@fastlane-bot the cert/sign steps are working completely fine

**Comment by olisaokonkwo** [comment]
---

**Comment by frrijn** [comment]
Is it just me that is having this problem? or are there more people? 
(still having this problem)

**Comment by waltermvp** [comment]
@frrijn where you able to figure this out?

**Comment by frrijn** [comment]
@waltermvp sadly not. Still having the same problem. Already spent quite some time trying to figure it out, but the error is just too general that I have no idea what it could be. But if you are able to find a solution. please let me know!:P

**Comment by waltermvp** [comment]
@frrijn same here manually uploaded metadata. However take a look here, 
https://github.com/fastlane/fastlane/issues/8819#issuecomment-338058782

**Comment by basvankuijck** [comment]
I had a similar error. Turned out I was uploading `en-GB` metadata into an app that only had `English (U.S.)` active in iTunes connect. Renaming the `en-GB` to `en-US` fixed it for me.

**Comment by Salierik** [comment]
I have both languages for our apps (en-US and en-GB). And it is important for me - use both (Different prices in Description etc). And I have a problem in 1 my app. For other 7 is working fine.

There is a stack-trace:
`[⠇] Activating languages da, de-DE, el, en-GB, en-US, es-ES, es-MX, fi, fr-FR, fr-ca, id, it, ja, ko, ms, nl-NL, no, pt-BR, 
pt-PT, ru, sv, th, tr, vi, zh-Hans, zh-Hant...                                                                              
Looking for related GitHub issues on fastlane/fastlane...                                                                   
Search query: The request could not be completed because:                                                                   
        Unexpected Error`

`/home/aida/.rvm/gems/ruby-2.3.1/gems/fastlane-2.77.1/spaceship/lib/spaceship/tunes/tunes_client.rb:225:in `handle_itc_respon
se': [!] The request could not be completed because: (Spaceship::Tunes::PotentialServerError)                               
        Unexpected Error                                                                                                    
        from /home/aida/.rvm/gems/ruby-2.3.1/gems/fastlane-2.77.1/spaceship/lib/spaceship/tunes/tunes_client.rb:416:in `bloc
k in update_app_version!'                                                                                                   
        from /home/aida/.rvm/gems/ruby-2.3.1/gems/fastlane-2.77.1/spaceship/lib/spaceship/tunes/tunes_client.rb:1304:in `wit
h_tunes_retry'                                                                                                              
        from /home/aida/.rvm/gems/ruby-2.3.1/gems/fastlane-2.77.1/spaceship/lib/spaceship/tunes/tunes_client.rb:409:in `upda
te_app_version!'                                                                                                            
        from /home/aida/.rvm/gems/ruby-2.3.1/gems/fastlane-2.77.1/spaceship/lib/spaceship/tunes/app_version.rb:403:in `save!
'                                                                                                                           
        from /home/aida/.rvm/gems/ruby-2.3.1/gems/fastlane-2.77.1/deliver/lib/deliver/upload_metadata.rb:253:in `verify_avai
lable_languages!'                                                                                                           
        from /home/aida/.rvm/gems/ruby-2.3.1/gems/fastlane-2.77.1/deliver/lib/deliver/upload_metadata.rb:72:in `upload'     
        from /home/aida/.rvm/gems/ruby-2.3.1/gems/fastlane-2.77.1/deliver/lib/deliver/runner.rb:123:in `upload_metadata'    
        from /home/aida/.rvm/gems/ruby-2.3.1/gems/fastlane-2.77.1/deliver/lib/deliver/runner.rb:39:in `run'                 
        from /home/aida/.rvm/gems/ruby-2.3.1/gems/fastlane-2.77.1/deliver/lib/deliver/commands_generator.rb:79:in `block (2 
levels) in run'                                                                                                             
        from /home/aida/.rvm/gems/ruby-2.3.1/gems/commander-fastlane-4.4.5/lib/commander/command.rb:178:in `call'           
        from /home/aida/.rvm/gems/ruby-2.3.1/gems/commander-fastlane-4.4.5/lib/commander/command.rb:153:in `run'            
        from /home/aida/.rvm/gems/ruby-2.3.1/gems/commander-fastlane-4.4.5/lib/commander/runner.rb:476:in `run_active_comman
d'                                                                                                                          
        from /home/aida/.rvm/gems/ruby-2.3.1/gems/fastlane-2.77.1/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:75:i
n `run!'                                                                                                                    
        from /home/aida/.rvm/gems/ruby-2.3.1/gems/commander-fastlane-4.4.5/lib/commander/delegates.rb:15:in `run!'          
        from /home/aida/.rvm/gems/ruby-2.3.1/gems/fastlane-2.77.1/deliver/lib/deliver/commands_generator.rb:182:in `run'    
        from /home/aida/.rvm/gems/ruby-2.3.1/gems/fastlane-2.77.1/deliver/lib/deliver/commands_generator.rb:17:in `start'   
        from /home/aida/.rvm/gems/ruby-2.3.1/gems/fastlane-2.77.1/fastlane/lib/fastlane/cli_tools_distributor.rb:90:in `take
_off'                                                                                                                       
        from /home/aida/.rvm/gems/ruby-2.3.1/gems/fastlane-2.77.1/bin/fastlane:20:in `<top (required)>'                     
        from /home/aida/.rvm/gems/ruby-2.3.1/bin/fastlane:23:in `load'                                                      
        from /home/aida/.rvm/gems/ruby-2.3.1/bin/fastlane:23:in `<main>'                                                    
        from /home/aida/.rvm/gems/ruby-2.3.1/bin/ruby_executable_hooks:15:in `eval'                                         
        from /home/aida/.rvm/gems/ruby-2.3.1/bin/ruby_executable_hooks:15:in `<main>'`

**Comment by Salierik** [comment]
Guys, I've detected the problem's root: Fastlane is failing when have a dir with language who has not enabled in the Store. And failing when try to enable.

**Comment by nolanwilson** [comment]
>Guys, I've detected the problem's root: Fastlane is failing when have a dir with language who has not enabled in the Store. And failing when try to enable.

Fwiw, I also ran into this recently and was able to move beyond this issue by enabling the language in iTunes Connect first. And in case it's helpful, I also ran into the issue `The app name you entered is already being used.` even once I had activated the language. Maybe it was receiving that failure when it tried to activate the language in iTunes Connect which was what was throwing the `Unknown Error` message? It might not be related, but I wanted to mention it in case it helps with troubleshoot. We're also currently pinned on Fastlane 2.50.1.
